src/sql/azdata.proposed.d.ts

```diff
@@ -447,9 +446,13 @@ declare module 'azdata' {
 
 		export interface ConnectionDiagnosticsResult {
 			/**
-			 * Status indicating if the error was handled or not.
+			 * Property indicating if the error was handled or not.
```

Comment:
nit: We already know these are properties
```suggestion
			 * Whether the error was handled or not.
```